packages/contracts/src/EpochKeyProof.ts

```diff
@@ -0,0 +1,49 @@
+import { BigNumberish, utils } from 'ethers'
+import { SnarkProof } from '@unirep/crypto'
+import UnirepCircuit, { CircuitName } from '@unirep/circuits'
+
+import { UnirepTypes } from './contracts/IUnirep'
+import { rmFuncSigHash } from './utils'
+import { UnirepABI } from './abis/Unirep'
+
+export class EpochKeyProof implements UnirepTypes.EpochKeyProofStruct {
+    public globalStateTree: BigNumberish
```

Comment:
I think there's some value in taking a proof and unwrapping the property names like here, but I don't know if it's worth maintaining this code.

Instead could we improve the documentation for the proofs saying what each index in the proof is?

e.g.

```
EpochKeyProof:
[
  GST,
  epoch,
  epochKey,
]
```

This would let us get rid of these classes instead of having to maintain more code/docs.